INSTALLATION.md

```diff
@@ -55,7 +59,7 @@ After executing maven command, you will be requested to insert additional parame
   Naming recommendation is to use lower case letters and kebab-case.
 
 - **systemConnector1Package**/**systemConnector2Package** are used to define package name suffix for the connectors. Notice that
-connector package name starts with prefix defined on **package** step.
+  connector package name starts with prefix defined on **package** step.
```

Comment:
Space is not needed

Reply:
(somehow my first comment got lost, so don't wonder if you see it duplicated or something like that)

The space is necessary here, as this new line belongs to the list item created one line above. If newlines are used within list items the content on the new line needs to be indented at least as much as the text in the line starting the item